test/ModuleInterface/arm64e-fallback.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: not %target-typecheck-verify-swift -target arm64e-apple-ios13.0 -F %S/Inputs | %FileCheck %s
+
+// REQUIRES: OS=ios
+
+import DummyFramework
+// CHECK: could not find module 'DummyFramework' for target 'arm64e-apple-ios'; found: arm64-apple-ios, arm64
+
+public struct T {
```

Comment:
What’s the purpose of all of these declarations? Are they part of the test?

Reply:
Done, removed.